arch/arm/soc/nxp_kinetis/kwx/dts.fixup

```diff
@@ -0,0 +1,8 @@
+/* SoC level DTS fixup file */
+
+#if defined(CONFIG_SOC_MKW22D5) || defined(CONFIG_SOC_MKW24D5)
+#define FLASH_DEV_BASE_ADDRESS		NXP_K64F_FLASH_CONTROLLER_4001F000_BASE_ADDRESS_0
```

Comment:
This needs to be updated to FTFL

Reply:
Yeah, will fix that up when I introduce FTFL.